nginx.conf

```diff
@@ -469,5 +469,69 @@ location = /guides/manage-event-stream/ {
 }
 
 location = /performance-monitoring/setup/ {
-  return 302 /performance-monitoring/getting-started/;
-}
+  return 302 /product/performance/getting-started/;
+}
+  location = /workflow/alerts-notifications/ {
+    return 302 /product/alerts-notifications/;
+  }
+  
+  location = /workflow/alerts-notifications/alerts/ {
+    return 302 /product/alerts-notifications/alerts/;
+  }
+  
+  location = /workflow/alerts-notifications/notifications/ {
+    return 302 /product/alerts-notifications/notifications/;
+  }
+  
+  location = /workflow/issue-owners/ {
+    return 302 /product/error-monitoring/issue-owners/;
+  }
+  
+  location = /workflow/integrations/ {
+    return 302 /product/integrations/;
+  }
+   
+  location = /workflow/dashboards/ {
+    return 302 /product/error-monitoring/dashboards/;
+  }
+   
+  location = /workflow/debug-files/ {
+    return 302 /data-management/debug-files/;
+  }
+  
+  location = /workflow/releases/ {
+    return 302 /product/releases/;
+  }
+  
+  location = /workflow/search/ {
+    return 302 /product/sentry-basics/search/;
+  }
+  
+  location = /enriching-error-data/environments/ {
+    return 302 /product/sentry-basics/environments/;
+  } 
+  
+  location = /enriching-error-data/error-tracing/ {
+    return 302 /product/sentry-basics/guides/;
+  }
+  
+  location = /guides/ {
```

Comment:
should we even bother moving this? if ultimately these should live somewhere else (e.g. imagine another section similar to 'api docs' or 'developer docs') then changing the url now just to potentially change it in the future wont be very useful.

Reply:
We get a lot of asks from customers to provide best practices, and I've provided front matter to each of these guides for better context. 

I've moved it here to provide a primer to users as part of the concept of "basic info you should know". We can easily move it again if we think it needs a stronger location.